src/components/form-helpers/form-helpers.js

```diff
@@ -159,6 +159,8 @@ export const FormMessages = connect(
         ? { message: errors['_error'], className: 'dp-error' }
         : errors && errors['_warning']
         ? { message: errors['_warning'], className: 'dp-warning-message' }
+        : errors && errors['_success']
+        ? { message: errors['_success'], className: 'dp-ok-message' }
```

Comment:
Nice!, how does it look?

Reply:
![image](https://user-images.githubusercontent.com/8861133/61306493-5561c980-a7b2-11e9-8322-0fbbc593a215.png)